docs/notebooks/convcxn_train.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Tutorial: Set-up, create and train a convolutional CXN\n",
     "\n",
-    "In this notebook, we create and train a simplified, non-attentional version of a CXN network, originally proposed in the paper by Hajij et. al : Cell Complex Neural Networks (https://arxiv.org/pdf/2010.00743.pdf). We will load a cellular complex dataset from the web and train the model to perform classificaiton on this dataset."
+    "In this notebook, we create and train a simplified, non-attentional version of a CXN network, originally proposed in the paper by Hajij et. al : Cell Complex Neural Networks (https://arxiv.org/pdf/2010.00743.pdf). We will load a cell complex dataset from the web and train the model to perform classificaiton on this dataset."
```

Comment:
"non-attentional version of a CXN Network" --> "attentional CXN Network"

Reply:
Done. Also the notebooks/ folder shouldn't be a folder but a symlink. I've fixed that in another PR.